logstash-core/lib/logstash/config/loader.rb

```diff
@@ -0,0 +1,90 @@
+require "logstash/config/defaults"
+
+module LogStash; module Config; module Loader
```

Comment:
I think the scope of this module is out of the scope of this PR. If we talk about plugable agent, I don't understand the purpose of this loader config class here. Can you explain it to me?